securityheaders/use.go

```diff
@@ -0,0 +1,17 @@
+package securityheaders
+
+import "net/http"
+
+// Use applies common security headers to a handler.
+func Use(h http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("Content-Security-Policy", "default-src 'self'")
```

Comment:
If an app wanted to overwrite this, I guess they'd create their own middleware that wraps this one and use `w.Header().Set("Content-Security-Policy", "...")`?

Reply:
yeah, which is probably cleaner than giving this a bunch of options to override things